src/color.js

```diff
@@ -963,7 +977,7 @@ Object.assign(Color, {
 
 	// Global defaults one may want to configure
 	defaults: {
-		gamutMapping: "lch.chroma",
+		gamutMapping: "oklch.chroma",
```

Comment:
Not so sure about using this as a default, what happens if oklch is not available?

Reply:
(not saying we shouldn't have it as a default if oklch is, but that it shouldn't be the *only* default)